pkgs/top-level/all-packages.nix

```diff
@@ -20831,6 +20831,23 @@ with pkgs;
 
   zn_poly = callPackage ../development/libraries/science/math/zn_poly { };
 
+  ### SCIENCE/MATERIAL-SCIENCE
```

Comment:
Would you mind putting both packages into the category"science/chemistry"? There are already some electronic structure/DFT codes in that category.